Add split-video-scenes extension

remorses opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

pernielsentikaer commented on extensions/split-video-scenes/package.json:
```diff
@@ -0,0 +1,40 @@
+  "commands": [
+    {
+      "name": "index",
+      "title": "Split Video Into Scenes",
```
```suggestion
      "title": "Split Video into Scenes",
```

pernielsentikaer reviewed (changes requested):
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- Can we somehow improve this flow? I didn't have pip so I kinda stranded - is there a homepage we can land on instead for guiding the user?

I also tried with pip3 since it was installed but it didn't go

![Raycast26082024-V4neIoKj png@2x](https://github.com/user-attachments/assets/c0558d28-2092-45f6-b59a-8804c5c7f96e)

- I would also add a bit more to the `README` to explain what it does and which modes do what

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

remorses commented:
I updated the extension to install the command with pipx, so there is no longer the problem you faced. pipx creates a sandbox for each installed lib, so it does not create problem for the user python environment.

I also updated the code to detach the scenedetect process so an user can use other extensions while scenedetect is running (which can take minutes), if the user runs the extension while the process is in progress it will show the current logs. here is a demo

https://github.com/user-attachments/assets/60f3fe21-2634-4609-ad3d-a600456bd956

pernielsentikaer commented:
I think we might need to let the user handle this installation process. I ended up in a loop because something was wrong with my PATHs, and I was unable to see the message since the window closed. I had to record the screen in order to read it

![Raycast27082024-xDpga2sN png@2x](https://github.com/user-attachments/assets/f4e7bd0c-db39-4b21-b8df-9fe4cb326da9)

remorses commented:
I added the .local/bin folder in path, this should fix it permanently because pipx always installs packages there

pernielsentikaer commented on extensions/split-video-scenes/src/index.tsx:
```diff
@@ -0,0 +1,289 @@
+        canChooseDirectories={false}
+        canChooseFiles={true}
+        autoFocus
+        title="Select video files"
```
```suggestion
        title="Select Video Files"
```

pernielsentikaer commented on extensions/split-video-scenes/src/index.tsx:
```diff
@@ -0,0 +1,289 @@
+      exitCode = await new Promise((resolve) => childProcess!.on("close", (code) => resolve(code || 0)));
+    }
+    if (exitCode === 0) {
+      closeMainWindow();
```
Should we popToRoot and show a toast instead(with actions to open folder) - it felt weird that the window just closed without anything

remorses replied:
Now it shows an HUD and the folder with the scenes in finder

pernielsentikaer reviewed (changes requested):
That worked as expected 🔥

I gave it another spin and found a few things 🔥

pernielsentikaer reviewed (approved):
Hi 👋 

Looks good to me, approved 🔥
